cli/src/main/java/com/devonfw/tools/ide/version/VersionSegment.java

```diff
@@ -320,6 +324,60 @@ protected VersionLetters getDevelopmentPhase() {
     return result;
   }
 
+  /**
+   * {@link VersionIdentifier#incrementSegment(int, boolean)}  Increments a version} recursively per {@link VersionSegment}.
+   *
+   * @param segmentKeepCount the number of leading {@link VersionSegment}s to keep untouched. Will be {@code 0} for the segment to increment and negative
+   *     for the segments to set to zero.
+   * @param keepLetters {@code true} to keep {@link VersionSegment#getLetters() letters} from modified segments, {@code false} to drop them.
+   * @return the new {@link VersionSegment}.
+   */
+  public VersionSegment increment(int segmentKeepCount, boolean keepLetters) {
```

Comment:
Maybe provide some small examples for its usage here too. Not everyone might check the tests for its usage.